src/Styles/Header.css

```diff
@@ -0,0 +1,82 @@
+.panel-bg {
+  display: flex;
+}
+
+.panel-bg h1 {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.panel-bg ul {
+  display: flex;
+  list-style: none;
+  gap: 10px;
+
+  /* text-transform: uppercase; */
+  color: #121212;
+}
+
+.panel-bg ul li :first-child {
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.panel-bg ul li :last-child {
+  width: 6.563rem;
+  height: 1rem;
+
+  /* margin: 1.125rem 41.5rem 0.688rem 2.563rem; */
+  opacity: 0.5;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.Mask {
+  width: 1.063rem;
+  height: 1.063rem;
+  background-color: #0290ff;
+}
+
+/* .Oval {
+  width: 2.813rem;
+  height: 2.813rem;
+  margin: 0 0 0 41.5rem;
+  padding: 0.875rem;
+  border: solid 1px #571b1b;
+} */
```

Comment:
- [ ] Please remove all commented code from your codebase to make your code clean and easy to maintain.